tests/llm/utils/mock_toolset.py

```diff
@@ -657,6 +657,13 @@ def _initialize_toolsets(self):
         config_path = os.path.join(self.test_case_folder, "toolsets.yaml")
         custom_definitions = self._load_custom_toolsets(config_path)
 
+        # If no test-specific toolsets.yaml exists, use default toolsets.yaml
+        if not custom_definitions:
```

Comment:
Let's say I have the runbook tool enabled, and the toolset.yaml does not mention it.
Should the runbook tool be enabled in that case?
What happens here? and what do you think the behavior should be?